lib/clibuddy/runner.rb

```diff
@@ -197,10 +197,14 @@ def screen_working_width
       @screen_working_width ||= [TTY::Screen.width, 80].min
     end
 
-    def do_show_error(action)
+    def do_show_message(action)
```

Comment:
For show-message vs show-error I think it would be good to show the message without message id or special formatting in the case of the former.

Reply:
Never mind, I thought show-error included the error formatting + id. That shoudl probably be fixed, I'll open a new PR after this merges.